tests/bootstrap.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// define project root which will be used throughout the bootstrapping process
+define('PIMCORE_PROJECT_ROOT', dirname(__DIR__));
+
+const PROJECT_ROOT = PIMCORE_PROJECT_ROOT;
+
+// set the used pimcore/symfony environment
+putenv('APP_ENV=test');
+$_ENV['APP_ENV'] = 'test';
```

Comment:
I think we should set the env in `$_SERVER`, too.

```suggestion
$_ENV['APP_ENV'] = $_SERVER['APP_ENV'] = 'test';
```